warehouse/views.py

```diff
@@ -118,6 +124,7 @@ def httpexception_view(exc, request):
     response.headers.extend(
         (k, v) for k, v in exc.headers.items() if k not in response.headers
     )
+    response.headers.extend(_CORS_HEADERS)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** With this addition, `_CORS_HEADERS` should be applied to any `response` object in this function. ‏Are the other additions of `headers=_CORS_HEADERS` even necessary?

Reply:
Indeed. Thanks for the catch. I removed redundant `headers=_CORS_HEADERS`.